cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/common/Digest.java

```diff
@@ -19,25 +19,13 @@
 
 package org.apache.cassandra.spark.common;
 
-import java.security.MessageDigest;
-import java.util.Base64;
-
-public final class MD5Hash
+/**
+ * Interface that represents a checksum digest
```

Comment:
nit: checksum kind of equals to digest. It is repeat like "I have two animal cats". Checksum can be removed